.github/workflows/ci-include-breaking.yml

```diff
@@ -15,4 +15,11 @@ jobs:
       - name: Lint
         run: npm run lint 
       - name: Test
-        run: DID_WG_INCLUDE_BREAKING='🔥' npm run test 
+        run: DID_WG_INCLUDE_BREAKING='🔥' npm run test || true
+      - name: Build Report
+        run: npm run test-and-generate-report
+      - name: Deploy Report
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does @iherman need to add a GITHUB_TOKEN to the secrets for this repo?

Reply:
No, this variable is always available to github actions.

Reply:
ack... ty -- I remember having to set this up manually a long time ago in some GH actions... it was a pain... perhaps they added this to make sure no one had to ever do that again.